site/src/components/WorkspaceScheduleBanner/WorkspaceScheduleBanner.tsx

```diff
@@ -1,20 +1,28 @@
+import Button from "@material-ui/core/Button"
 import Alert from "@material-ui/lab/Alert"
 import AlertTitle from "@material-ui/lab/AlertTitle"
+import { useMachine } from "@xstate/react"
 import dayjs from "dayjs"
 import isSameOrBefore from "dayjs/plugin/isSameOrBefore"
 import utc from "dayjs/plugin/utc"
 import { FC } from "react"
 import * as TypesGen from "../../api/typesGenerated"
 import { isWorkspaceOn } from "../../util/workspace"
+import { workspaceScheduleBanner } from "../../xServices/workspaceSchedule/workspaceScheduleBannerXService"
```

Comment:
NIT: maybe we could alias this to differentiate it from the component.

Reply:
it already is: it's lowercased and the componented is uppercased. However, I am not opposed to tacking on the word `machine` at the end.

Reply:
I'm gonna go with `workspaceScheduleBannerMachine`. LMK if you think of a better (or shorter) name, but I think having the name be disambiguated by more than a capital letter is really helpful conceptually and for navigating code. I definitely don't want to dismiss a legitimate naming concern. My prior comment wasn't written very well, what I mean to say is there wasn't name collision due to the capital letter, but I think that's not satisfactory enough

Reply:
I made the change here: https://github.com/coder/coder/pull/1987/commits/650aa4ea8b8a7e797e6ceade0986f070454a231f